src/lib/types.ts

```diff
@@ -70,6 +70,17 @@ export interface RegExpFilter {
   obj: RegExp;
   source: string;
 }
+
+export type Filters = {
+  annotations: Record<string, string>;
+  deletionTimestamp: boolean;
+  labels: Record<string, string>;
+  name: string;
+  namespaces: string[];
+  regexName: string | RegExp;
+  regexNamespaces: RegExp[]; // TODO: Test the Regexp path??
```

Comment:
Why are we changing the types of these?

Reply:
It turns out we actually shouldn't do this. We'll revert that particular change before we mark the PR ready for review.